modules/schematics/src/feature/schema.json

```diff
@@ -56,6 +56,12 @@
       "description":
         "Specifies if api success and failure actions, reducer, and effects should be generated as part of this feature.",
       "aliases": ["a"]
+    },
+    "effectCreator": {
+      "type": "boolean",
+      "default": false,
+      "description": "Specifies if the effect creation uses 'createEffect'",
+      "aliases": ["ec"]
```

Comment:
Same here

Reply:
I think we should simplify here and just use `--creators`  in the `feature` schema to use the new syntax for functions for actions and effects. Its unlikely that you'll use one without the other, so let's shift people in that direction.